deno.json

```diff
@@ -0,0 +1,10 @@
+{
+  "imports": {
+    "@deno/dnt": "jsr:@deno/dnt@^0.42.1",
+    "@std/assert": "jsr:@std/assert@^1.0.13",
+    "@std/cli": "jsr:@std/cli@^1.0.17"
+  },
+  "compilerOptions": {
+    "experimentalDecorators": true
+  }
```

Comment:
tsconfig.json is not needed now!!

Reply:
Do we want experimentalDecorators on again?

What it offers changes release by release.

We could try targeting `ES7` which would be pretty stable and we wouldn't have to worry for a while.

Reply:
see the discussion https://github.com/GameBridgeAI/ts_serialize/discussions/158